packages/graphql/tests/tck/undirected-relationships/query-direction-aggregations.test.ts

```diff
@@ -95,7 +95,7 @@ describe("QueryDirection in relationships aggregations", () => {
         expect(formatParams(result.params)).toMatchInlineSnapshot(`"{}"`);
     });
 
-    test("query with a UNDIRECTED_ONLY relationship", async () => {
+    test("query with a UNDIRECTED_ONLY (deprecated) relationship", async () => {
```

Comment:
could it be worth moving these tests to a `deprecated` folder? so we remember to remove them later

Reply:
Done!